src/var/isAttached.js

```diff
@@ -0,0 +1,10 @@
+define( function() {
+	"use strict";
+
+	return function isAttached( obj ) {
+
+      // Replace the use of contains function to check attachment (gh-3504)
```

Comment:
This comment is not necessary, at least not in this form. We don't need to describe what the code looked like previously or link to the issue/PR that changed it.

Reply:
Okay. Will remove it for the moment.